service/src/main/resources/application.yml

```diff
@@ -55,6 +55,19 @@ cbas:
     # How often to check downstream services for health:
     healthCheckIntervalSeconds: 300
 
+management:
```

Comment:
Hi @cjllanwarne -- I'd recommend expanding this configuration slightly to also get percentile histogram metrics for endpoint latencies.  More info here: 

https://github.com/DataBiosphere/terra-workspace-manager/pull/1607

Reply:
Cool!!

Reply:
This seems to be prometheus specific? It doesn't look like these show up in the default `AppRequests` section of the azure console?